core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -4115,7 +4131,8 @@ class ReplicaManagerTest {
           safeYammerMetricValue("type=DelayedRemoteFetchMetrics,name=ExpiresPerSec").asInstanceOf[Long])
       latch.countDown()
     } finally {
-      replicaManager.shutdown(checkpointHW = false)
+      CoreUtils.swallow(replicaManager.shutdown(checkpointHW = false), this)
```

Comment:
With the suggested change of using [Utils.tryAll()](https://github.com/apache/kafka/blob/810f63b5b645666c004a6a1222eeed54b202c10e/clients/src/main/java/org/apache/kafka/common/utils/Utils.java#L1197) instead of swallowing exceptions, it becomes 

```
    Utils.tryAll(util.Arrays.asList[Callable[Void]](
        () => {
          replicaManager.shutdown(checkpointHW = false)
          null
        },
        () => {
          remoteLogManager.close()
          null
        },
      ))
```

Reply:
I agree. Will update it soon.

Reply:
Ah nice utility. TIL.